error-handling/src/test/java/com/palantir/conjure/java/errors/SerializableErrorErrorDecoderTests.java

```diff
@@ -136,7 +136,7 @@ public void testNullBody() {
     public void testRemoteExceptionIgnoresUnknownProperties() throws Exception {
         String stackTrace = "{\"methodName\" : \"methodName\", \"fileName\" : \"fileName\", \"lineNumber\" : 0,"
                 + "\"className\" : \"className\" , \"noSuchProperty\" : false }";
-        String error = "{\"message\": \"message\", \"exceptionClass\": \"exceptionClass\","
+        String error = "{\"errorCode\": \"code\", \"errorName\": \"foo\","
```

Comment:
Yeah, sorry. Planned to remove deprecations in another PR, but since this PR picked up https://github.com/palantir/http-remoting-api/pull/100, where the old fields had already been removed, I had to update this too.

The working assumption was that remoting2 servers are no longer supported by conjure-java, but they will always be supported by remoting3, and these two can coexist as long as necessary.